README.md

```diff
@@ -1,24 +1 @@
-# briefjcb-vue
-
-## Project setup
-```
-npm install
-```
-
-### Compiles and hot-reloads for development
-```
-npm run serve
-```
-
-### Compiles and minifies for production
-```
-npm run build
-```
-
-### Lints and fixes files
-```
-npm run lint
-```
-
-### Customize configuration
-See [Configuration Reference](https://cli.vuejs.org/config/).
+development branch for vue briefjcb
```

Comment:
in general for any new projects, it's good to add some description and instructions on steps on how to run the project locally